frontend/pulse3d/src/pages/uploads.js

```diff
@@ -130,23 +129,22 @@ export default function Uploads() {
   const [pending, setPending] = useState(true);
   const [filterString, setFilterString] = useState("");
   const [filterColumn, setFilterColumn] = useState("");
-  const [ownerWidth, setOwnerWidth] = useState("180px");
-  const [recordingWidth, setRecordingWidth] = useState("300px");
-  const [uploadWidth, setUploadWidth] = useState("250px");
-  const [createdWidth, setCreatedWidth] = useState("250px");
-  const [analyzedWidth, setAnalyzedWidth] = useState("200px");
-
+  const [ownerWidth, setOwnerWidth] = useState("10%");
+  const [recordingWidth, setRecordingWidth] = useState("30%");
+  const [uploadWidth, setUploadWidth] = useState("23%");
+  const [createdWidth, setCreatedWidth] = useState("15%");
+  const [analyzedWidth, setAnalyzedWidth] = useState("15%");
+  const [sortColumn, setSortColumn] = useState("");
   const uploadTableColumns = [
     {
-      width: "30px",
+      width: "3%",
       admin: false,
       cell: (row) => (
         <Checkbox
           id={row.id}
-          disabled={row.jobs.filter((job) => job.status !== "finished").length > 0}
           checked={checkedUploads.includes(row.id)}
           onChange={handleCheckedUploads}
-          style={{ width: "1px", margin: "0" }}
+          disabled={row.jobs.filter((job) => job.status == "pending").length > 0}
```

Comment:
Sorry I think I misspoke on this, the individual jobs are disabled as expected with `disabled={job.status === "pending"}`, so we don't want any restrictions here on the actual uploads.. so what you had was right. We just want to make sure that the modal pops up as expected if they select an upload and there is a pending/error jobs

Reply:
I added this back in. The modal seems to be working but I will check in my errors pr

Reply:
We just want to make sure the corresponding modal pops up if:
-  they select all and the upload has a pending job if attempting to download or delete 
-  they select all and has an error job if attempting to download

Reply:
I see. Just confirmed modal does pop up in both situations. A count of succeeded files is shown too.

Reply:
Actually, I don't think there is a modal for the first case.. so does something pop up?
I guess is fine since the checkbox is clearly disabled. Might want to prevent deleting of an entire upload tree if it contains a pending job because I'm not sure what will happen if they try. Probably just the job would error if it attempts to update the jobs_result table and the entry is gone

Reply:
No matter what you select, it will only download completed jobs. If there are none then it will tell the user no valid files to download/  delete selected.

Reply:
Okay cool, sorry just wanted to double check. So if a user selects an upload and all jobs, but it contains a pending job, it won't let them delete correct?

Reply:
It does let them delete. I think I see what your saying now. If there is a pending job, it should error and not take action?

Reply:
or remove the pending jobs and still take action?

Reply:
We don't want them to be able to delete the upload that contains a pending job

Reply:
Can I add it to this ticket? https://app.asana.com/0/1152302614804188/1203331640504342/f
This ticket is about  fixing some bugs with alert modals.